engine/src/main/java/org/terasology/input/cameraTarget/PlayerTargetSystem.java

```diff
@@ -56,6 +58,14 @@ public EntityRef getTarget() {
         return targetSystem.getTarget();
     }
 
+    /**
+     * Get the position of the block that is currently targeted.
+     * @return the position of the block, a vector.
+     */
+    public Vector3i getTargetBlockPosition() {
+        return targetSystem.getTargetBlockPosition();
+    }
```

Comment:
This is related to the comment I made directly below this - I'm having to bend over backwards to avoid calling `getTarget()` from `CharacterSystem` to get around a presumed bug in the entity system.